### PR TITLE
Use Throwable.addSuppressed

### PR DESCRIPTION
**Pull request by jglick**
@reviewbybees

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/workflow/steps/BodyExecutionCallback.java [hunk 1]
```diff
@@ -112,19 +112,20 @@ public void onFailure(StepContext context, Throwable t) {
         @Override public final void onSuccess(StepContext context, Object result) {
             try {
                 finished(context);
-                context.onSuccess(result);
```
Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/workflow/steps/BodyExecutionCallback.java]: No need to catch exceptions from `onSuccess` itself; these should not happen, but would be thrown up the call chain.

**Comment by abayer** [comment]
:bee:

**Comment by svanoort** [comment]
🐝

**Comment by jglick** [comment]
cf. [JENKINS-30848](https://issues.jenkins-ci.org/browse/JENKINS-30848)

**Pull request merged**